release/release.js

```diff
@@ -46,8 +46,8 @@ const masterBranchDefault = "master2"
 const devBranchDefault = "dev2"
 const docBranchDefault = "gh-pages2"
 const versionReleasedDefault = VERSION
-const versionCodeReleasedDefault = 65
-const nextVersionDefault = "7.3.0"
+const versionCodeReleasedDefault = 66
+const nextVersionDefault = "8.0.0"
```

Comment:
@wmathurin Isn't the next release `7.3.0`? Or is this meant to be the `n+2` release?

Reply:
constants.js has a VERSION field (which is the release being worked on - and that release.js is trying to produce). After the release is cut, VERSION will be updated to nextVersion in dev.

Reply:
Ah, I see.